src/frontend/komponenter/Behandling/Vurdering/EndreVurdering.tsx

```diff
@@ -13,44 +13,54 @@ const StyledEndreVurdering = styled.div`
 `;
 
 interface Props {
-    config: IVilkårConfig;
     data: IVurdering;
+    inngangsvilkår: IInngangsvilkår;
     lagreVurdering: (vurdering: IVurdering) => Promise<Ressurs<string>>;
     settRedigeringsmodus: (erRedigeringsmodus: boolean) => void;
 }
 
-const EndreVurdering: FC<Props> = ({ config, data, lagreVurdering, settRedigeringsmodus }) => {
+const EndreVurdering: FC<Props> = ({
+    data,
+    lagreVurdering,
+    settRedigeringsmodus,
+    inngangsvilkår,
+}) => {
     const [vurdering, settVurdering] = useState<IVurdering>(data);
     const [feilmelding, setFeilmelding] = useState<string | undefined>(undefined);
     const [oppdatererVurdering, settOppdatererVurdering] = useState<boolean>(false);
 
     const oppdaterVurdering: () => void = () => {
         settOppdatererVurdering(true);
-        lagreVurdering(vurdering)
-            .then((ressurs) => {
-                if (ressurs.status === RessursStatus.SUKSESS) {
-                    setFeilmelding(undefined);
-                    settRedigeringsmodus(false);
+        setFeilmelding(undefined);
+        settRedigeringsmodus(false);
+        lagreVurdering(vurdering).then((ressurs) => {
+            if (ressurs.status !== RessursStatus.SUKSESS) {
```

Comment:
Hva skjer nå hvis det ikke er suksess fra backend?

Reply:
Det er 2 saker som skjer med denne endringen:
Hvis backend feiler, så renderer den "visVurdering" virker det som, men så renderer den EndreVurdering rett etterpå, i tillegg så viser den ikke noen feilmelding hvis backend feiler.

Den renderer de nye verdiene etter att man mottatt lagringen fra backend (tidligere ble den ikke rendert før backend returnerte noe)
![renderer_verdier_etter_lagring](https://user-images.githubusercontent.com/937168/101220962-f5a9de00-3686-11eb-90b4-c77777862eb2.gif)

Hvis det feiler (tidligere ble det vist en feilmelding)
![lagring_feiler](https://user-images.githubusercontent.com/937168/101221006-14a87000-3687-11eb-8fa5-62f69da08223.gif)

Reply:
Godt poeng - det er et problem at logikken for sukess/fail ligger på forskjellige steder. Vi prøver å flytte alt til "barnekomponenten"

Reply:
Bra catch, vi flyttet all håndtering av oppdatering opp til "mor"-komponenten ettersom barnekomponentene skal være gjenbrukbare. 
Barnekomponenten må derimot selv stå for oppdatering av visning/redigeringsmodus